spec/httparty/response_spec.rb

```diff
@@ -39,10 +39,16 @@
     it "should set code" do
       expect(@response.code).to eq(@response_object.code)
     end
-
+        
     it "should set code as an Integer" do
       expect(@response.code).to be_a(Integer)
     end
+    
+    it "should set http_version" do
+      unparseable_body = lambda { raise "Unparseable" }
+      unparseable_response = HTTParty::Response.new(@request_object, @response_object, unparseable_body)
+      expect(unparseable_response.http_version).to eq(@response_object.http_version)
```

Comment:
It seems that the following:

```ruby
expect(unparseable_response.response.http_version).to eq(@response_object.http_version)
```

Also works fine. The problem is that I can understand how delegation would be expected to work as well. As you said, it doesn't because parsed_response is checked first in method missing. I'm tempted to change the order, but no idea how severe of a change that would be, so I'll just merge this.

Reply:
:+1: